data/brands/shop/pawnbroker.json

```diff
@@ -10,6 +10,15 @@
   },
   "items": [
     {
+      "displayName": "Ablaza Pawnshop",
+      "locationSet": {"include": ["ph"]}
+      "tags": {
+        "brand": "Ablaza Pawnshop",
+        "brand:wikidata": "",
```

Comment:
We need to add the Wikidata for the brand.

Reply:
Ok

Reply:
Oopsie. That's an oversight on my part. Thanks @tommylung